sdk/monitor/opentelemetry-exporter/.prettierrc.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
again, should be able to leverage a common config

Reply:
I just deleted this and I think it picks up the common one automatically. Nice!